src/main/java/org/openlmis/requisition/service/RequisitionStatusNotifier.java

```diff
@@ -178,18 +178,6 @@ private FacilityDto getFacility(Requisition requisition) {
     return facilityReferenceDataService.findOne(requisition.getFacilityId());
   }
 
-  private String getStatus(Requisition requisition) {
```

Comment:
It's great that we get rid of that method. It was definitely a temporary solution.